modules/apps/friendly-url/friendly-url-test/src/testIntegration/java/com/liferay/friendly/url/service/test/FriendlyURLEntryLocalServiceTest.java

```diff
@@ -490,6 +491,14 @@ public void testValidateUrlTitleWithInvalidLength() throws Exception {
 			_group.getGroupId(), classNameId, urlTitle);
 	}
 
+	@Test(expected = InvalidFriendlyURLException.class)
```

Comment:
This will need to be updated to the new exception class.

Reply:
@jonathanmccann thanks for catching this!